sdv/io/local/local.py

```diff
@@ -125,15 +125,25 @@ def read(self, folder_name, file_names=None):
             file_paths = [folder_path / file for file in file_names]
 
         # Read CSV files
+        kwargs = {
+            'sep': self.sep,
+            'encoding': self.encoding,
+            'parse_dates': False,
+            'low_memory': False,
+            'decimal': self.decimal,
+            'on_bad_lines': 'warn'
+        }
+
+        args = inspect.getfullargspec(pd.read_csv)
+        if 'on_bad_lines' not in args.kwonlyargs:
+            kwargs.pop('on_bad_lines')
+            kwargs['error_bad_lines'] = False
```

Comment:
is this because of a old pandas version?

Reply:
Yeah, minimum test fails because of that.